sources/platform/actors/running/usage_and_resources.md

```diff
@@ -42,87 +52,105 @@ Sometimes, you see the Actor's CPU use go over 100%. This is not unusual. To hel
 
 ### Disk
 
-The Actor has hard disk space limited by twice the amount of memory. For example, an Actor with 1024 MB of memory will have 2048 MB of disk available.
+The Actor has hard disk space limited by twice the amount of memory. For example, an Actor with `1024MB` of memory will have `2048MB` of disk available.
 
 ## Requirements
 
-Actors built on top of the [Apify JS SDK](/sdk/js) and [Crawlee](https://crawlee.dev/) use autoscaling. This means that they will always run as efficiently as they can with the memory they have allocated. So, if you allocate 2 times more memory, the run should be 2 times faster and consume the same amount of compute units (1 * 1 = 0.5 * 2). Autoscaling for Python is not yet available, but it is planned for the near future.
+Actors built with [Apify JS SDK](/sdk/js) and [Crawlee](https://crawlee.dev/) use autoscaling. This means that they will always run as efficiently as they can based on the allocated memory. So, if you double the allocated memory, the run should be twice as fast and consume the same amount of compute units (1 * 1 = 0.5 * 2).
```

Comment:
This should be checked with someone from delivery / tooling. Not sure if this is 100% true, as it contradicts a bit the cheerio section later (with mention of node threads)

Reply:
it feels correct. 4g memory = 1cpu core, so the note about max memory of 4g for cheerio makes sense, the node process will only use a single thread there (sidenote: that's how we have it implemented, we could try to leverage worker threads or other similar parallelization features of node to get around this). 

cc @metalwarrior665 for field experience :]

Reply:
1. Yes, 4 GB is a target for non-browser Node.js actors because of the single core restriction. 

2. Also, Apify SDK doesn't use any autoscaling, there is nothing to scale there, that is only related to Crawlee.

Reply:
So, as I understand, mention of Apify JS SDK should be removed from this paragraph, as it is only regarding Crawlee?

Reply:
Yep.